etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/SwapPlacerOrderAndGroupNumbersTest.groovy

```diff
@@ -34,8 +34,8 @@ class SwapPlacerOrderAndGroupNumbersTest extends Specification {
         expect:
         orc2_1 == "423787478"
         orc2_2 == "EPIC"
-        orc4_1 == "57128-1"
-        orc4_2 == "Newborn Screening Report summary panel"
+        orc4_1 == null
+        orc4_2 == null
```

Comment:
These should now be `null`?

Reply:
The MN example we're using in this test doesn't have any values in ORC-4. None of the ORU examples we have has it. I tested locally adding those values but was not sure if I should update that example because it would modify the sample we got from MN